Update issue templates (#8939)
kyma-project/kyma: .github/ISSUE_TEMPLATE/consulting.md

---

> name: Consulting

Reviewer: How about a **Support request** instead, similarly to what [k8s](https://github.com/kubernetes/kubernetes/issues/new/choose) has? Personally, I like this wording better.

> about: Ask a question or request for consultancy

Reviewer: ```suggestion
about: Request support or ask a question relating to Kyma
```
